site/content/en/docs/tasks/manage/setup_failure_recovery.md

```diff
@@ -0,0 +1,211 @@
+        resources:
+          requests:
+            cpu: "1"
+            memory: "200Mi"
+      restartPolicy: Never
+```
+
+Create the job:
+
+```sh
+kubectl create -f sample-job.yaml
+```
+
+And find the node its pod is scheduled on:
+
+```sh
+kubectl get pods -o wide
+```
+
+```
+NAME                     READY   STATUS    RESTARTS   AGE   IP            NODE           NOMINATED NODE   READINESS GATES
+sample-job-l8pts-qcwbc   1/1     Running   0          4s    10.244.2.29   kind-worker2   <none>           <none>
+```
+
+In this particular case, the pod is scheduled on a node called `kind-worker2`.
+
+## 2. Disable the `kubelet` service in the node
+
+The easiest way to simulate a node outage is to open a shell with `ssh` and disable the `kubelet` service.
+
+In the case of a `kind` cluster, a shell can be opened with:
+```sh
+docker exec -it kind-worker2 bash
+```
+
+To stop the `kubelet` service, run:
+```sh
+systemctl stop kubelet
+```
+
+This will cause the node to stop sending health pings to the control plane.
+
+## 3. Observe the node being down
+
+The `kubelet` on worker nodes periodically sends heartbeat pings to the control plane to report their health.
+If this this does not happen for `node-monitor-grace-period` (by default 50 seconds), the node's readiness status will be
+deemed `Unknown` and a `node.kubernetes.io/unreachable` taint will be added to the node.
+
+After waiting for the `node-monitor-grace-period` to elapse and running:
+```sh
+kubectl get nodes
+```
+the output should report the node's unreachability.
+```
+NAME                 STATUS     ROLES           AGE   VERSION
+kind-control-plane   Ready      control-plane   31h   v1.34.0
+kind-worker          Ready      <none>          31h   v1.34.0
+kind-worker2         NotReady   <none>          31h   v1.34.0
+```
+
+## 4. Observe the pod being terminated
+
+By default, pods have a 5 minute toleration for the `node.kubernetes.io/unreachable` taint.
+As such, for the first 5 minutes after the node status was reported as `NotReady` (and the taint was added), the
+pod should continue running.
+
+After the 5 minute grace period elapses, the pod will be marked for termination:
+```sh
+kubectl get pods
+```
+Showing:
+```
+NAME                     READY   STATUS        RESTARTS   AGE
+sample-job-l8pts-qcwbc   1/1     Terminating   0          6m16s
+```
+
+Without the failure recovery policy, the pod will be unable to progress past this point and will be stuck in the
+`Terminating` status until the node goes back online or the administrator intervenes manually.
+
+## 5. Observe the pod transition to `Failed` and a replacement being scheduled
+
+By default, pods have a 30 second grace period to terminate gracefully.
+Additionally, the failure recovery controller also waits 60 seconds before forcefully marking the stuck pod as `Failed`.
+In total, the pod should transition to `Failed` after 90 seconds since the `deletionTimestamp` was added to the pod.
```

Comment:
Please note that `deletionTimestamp` is mentioned also here.
(And now, after you followed my other comment, it's even more "out of the blue" in this place :)